### ISSUE TITLE
Sorting by Tag

### ISSUE THREAD

**Issue by ShinyRobotGod** [X2CommunityCore/xcom2-launcher#240]
Sorting by Tag doesn't sort by Tag.  Examples attached.
![AMM Tag Sorting 1](https://user-images.githubusercontent.com/60163496/100009846-a01f2880-2d8c-11eb-8356-afdb9bfe0a01.png)
![AMM Tag Sorting 2](https://user-images.githubusercontent.com/60163496/100009888-b0370800-2d8c-11eb-8720-35704e68dc2d.png)
![AMM Tag Sorting 3](https://user-images.githubusercontent.com/60163496/100009899-b2996200-2d8c-11eb-8788-463bd50ba2d8.png)

**Comment by RevZero** [comment]
Hmm, that could probably be improved. But sorting by tags doesn't make that much sense as soon as you have multiple tags assigned. It would stop to make sense depending on tag order.

You may want to use groups/categories instead. It probably makes more sense than tags, with respect to the tag categories you are using.

**Comment by ShinyRobotGod** [comment]
Yes, I can see your point that you might want multiple tags on the same mod like "Cosmetic" and "Star Wars", so maybe not the best method.  I created "categories" with the same names and that sorted just fine.  Thanks.
